scripts/delete_kibana_indices.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Set the default config file
+DEFAULT_CONFIG_FILE="./config/kibana.dev.yml"
```

Comment:
I think the path is missing one dot
```suggestion
DEFAULT_CONFIG_FILE="../config/kibana.dev.yml"
```